cmd/src/debug.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var debugCommands commander
+
+func init() {
+	usage := `'src debug' gathers and bundles debug data from a Sourcegraph deployment for troubleshooting.
+
+Usage:
+
+	src debug command [command options]
+
+The commands are:
+
+	kube                 generates kubectl outputs
+	comp                 generates docker outputs
```

Comment:
`comp` -> `compose` seems like a better, more explicit subcommand. Don't worry about reducing typing here! I wondered if `comp` was short for `computer` here, weirdlyi

Reply:
But all the subcommands are four letters! I do see what you're saying though, compose is much more explicit, I literally just wanted the four letters.